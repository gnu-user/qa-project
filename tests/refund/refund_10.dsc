TITLE
    Refund Amount Input (Invalid input, negative value)
DESCRIPTION
    Run the login transaction using an admin account. Initiate the refund 
    transaction. When prompted, enter a buyer which exists. When prompted, 
    enter a seller which exists. When prompted, enter a negative decimal value 
    for the refund amount. Verify that the value is rejected and an error 
    occurs.
DEPENDENCIES
    refund_7
