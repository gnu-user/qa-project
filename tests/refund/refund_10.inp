login
administrator
refund
buyer
seller
-10.00