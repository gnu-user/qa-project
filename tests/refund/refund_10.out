Welcome.
> 
Enter username: 
Username accepted.
Available tickets file read successfully.
>
Enter the buyer's username: 
Enter the seller's username: 
Enter the amount of credit to transfer: 
Invalid credit amount, amount cannot be less than 0.00.